engine/graphics/src/opengl/graphics_opengl.cpp

```diff
@@ -543,7 +569,7 @@ static void LogFrameBufferError(GLenum status)
 
     static bool OpenGLIsSupported()
     {
-        return (glfwInit() == GL_TRUE);
+        return true;
     }
```

Comment:
If we got this far, we already know we have opengl support

Reply:
Then I guess this function is more on the deprecated side of things?
If we do checks before this step, then we'll likely never really call the "IsSupported()" function?

Reply:
No, we still need it for other adapters (vulkan + dx12 afaik).